harness/determined/tensorboard/base.py

```diff
@@ -83,6 +109,21 @@ def delete(self) -> None:
         """
         pass
 
+    def start_async_upload_thread(self) -> None:
```

Comment:
I recommend just calling this "start", since I think, like "close", people are comfortable enough with the idea that some things need starting and closing without the name reflecting exactly _what_ gets started.

From a naming perspective, I think a bare "start()" call actually makes it clearer that this isn't a normal method, and that it is one that _should_ be called shortly after initialization, rather than just a method you should call when you think you need what it does.